plugins/modules/flightctl_approve.py

```diff
@@ -87,19 +87,9 @@ def main():
         argument_spec=argument_spec,
     )
 
-    kind = module.params.get("kind")
-    name = module.params.get("name")
-    params = {}
-    if module.params.get("approved"):
-        params["approved"] = module.params["approved"]
-    if module.params.get("approved_by"):
-        params["approvedBy"] = module.params["approved_by"]
-    if module.params.get("labels"):
-        params["labels"] = module.params["labels"]
-
     # Attempt to approve the request identified by name
     try:
-        perform_approval(module, kind, name, params)
```

Comment:
I will probably suggest that you think of a better name for this module, since it can approve or deny a request. Something like `flightctl_enrollment`.

Reply:
I agree, I modified the module name to be `flightctl_certificate` as the common functionality between the resources used by the module is approving / denying certificates and I could see it wrapping  other certificate-based functionality in the future.